src/setup.py

```
@@ -0,0 +1,37 @@
+from pathlib import Path
```

Pylint (4.0.3)

[convention] Missing module docstring

(C0114)